data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -344,45 +445,61 @@ def _health_factor(self) -> bool:
         # of households where household income is less than or equal to twice the federal
         # poverty level. Source: Census's American Community Survey]
 
-        health_criteria = (
-            (
-                self.df[
-                    field_names.DIABETES_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.ASTHMA_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.HEART_DISEASE_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
-            | (
-                self.df[
-                    field_names.LIFE_EXPECTANCY_FIELD
-                    + field_names.PERCENTILE_FIELD_SUFFIX
-                ]
-                # Note: a high life expectancy is good, so take 1 minus the threshold to invert it,
-                # and then look for life expenctancies lower than that (not greater than).
-                <= 1 - self.ENVIRONMENTAL_BURDEN_THRESHOLD
-            )
+        health_eligibility_columns = [
+            field_names.DIABETES_LOW_INCOME_FIELD,
+            field_names.ASTHMA_LOW_INCOME_FIELD,
+            field_names.HEART_DISEASE_LOW_INCOME_FIELD,
+            field_names.LIFE_EXPECTANCY_INCOME_FIELD,
```

Comment:
needs to be field_names.LIFE_EXPECTANCY_LOW_INCOME_FIELD

Reply:
in all two places you use it

Reply:
Yes that should already addressed